spack/prod/var/spack/environments/prod/spack.yaml

```diff
@@ -0,0 +1,91 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - cmake
+  - r@4.5.1
+  - gcc@13.4.0
+  - blas
+  - openblas
+  - libmd
+  - libbsd
+  - libedit
+  - bison
+  - python@3.12.9
+  - python@3.13.5
+  - which
+  - texinfo
+  - texlive
+  - libtirpc
+  - icu4c
+#  - openmpi@4.1.8+cuda fabrics=ucx schedulers=slurm
+  - openmpi@5.0.8+cuda fabrics=ucx schedulers=slurm
+  - cuda@11.8.0
+  - cuda@12.9.0
+  - go
+  - mpich@4.3.0+cuda+slurm
+  - apptainer@1.4.1~suid
+  view:
+    prod:
+      root: /software/spack/prod/view
+      projections:
+        all: "{name}-{version}-{compiler.name}-{compiler.version}-{hash}"
+  concretizer:
+    unify: when_possible
+  mirrors:
+    cache: file:///software/spack/cache
+  packages:
+    all:
+      target: [x86_64_zen4]
```

Comment:
Specify gcc 13.4.0 by default for all packages as described here?

https://spack.readthedocs.io/en/latest/packages_yaml.html#setting-default-requirements